Source/HYPScatterPlot.m

```diff
@@ -34,6 +34,10 @@ @interface HYPScatterPlot()
 
 @property (nonatomic) UIView *plotView;
 
+@property (nonatomic) UILongPressGestureRecognizer *panGestureRecognizer;
```

Comment:
@3lvis It is the same gesture recognizer for both the initial long pressure and the panning, have any genius suggestion for namning?